### PR TITLE
Fixes 3.1

### PR DESCRIPTION
**Pull request by delarosatrevin**
### Fixes:

* Allow motioncor  to fail for some movies without crashing the whole protocol
* Fixed several summaries (improved summary for ctf-refinement
* Fixed missing labels in convert (read/write)
* ~~Extend extract - particles to allow input partices and preserve particles properties~~

PS: will address the extract particles extension in another PR

### CODE REVIEW

File: relion/convert/convert31.py [hunk 1]
```diff
@@ -429,6 +434,7 @@ def writeSetOfParticles(self, partsSet, starFile, **kwargs):
             partsWriter.writeTableName('particles')
             partsWriter.writeHeader(partsTable.getColumns())
             # Write all rows
+            from pprint import pprint
```
Comment (azazellochg) [inline relion/convert/convert31.py:437]: this is not used

File: relion/tests/test_convert_relion.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 import os
 import subprocess
 import numpy as np
+from pprint import pprint
```
Comment (azazellochg) [inline relion/tests/test_convert_relion.py:33]: this is not used?

File: relion/tests/test_convert_relion.py [hunk 3]
```diff
@@ -781,6 +790,44 @@ def test_readSetOfParticles(self):
         self.assertEqual(fog.rlnMtfFileName, 'mtf_k2_200kV.star')
         self.assertEqual(fog.rlnOpticsGroupName, 'opticsGroup1')
 
+    def test_readSetOfParticlesAfterCtf(self):
+        if not Plugin.IS_GT30():
+            print("Skipping test (required Relion > 3.1)")
```
Comment (azazellochg) [inline relion/tests/test_convert_relion.py:795]: > 3.0 or >=3.1

**Review by azazellochg — APPROVED** [review]
I have looked at the changes but not tested, looks good.

File: relion/tests/test_convert_relion.py (hunk 3 above)
Comment (delarosatrevin) [reply relion/tests/test_convert_relion.py:795]: I will keep the tests for >=3.1

**Pull request merged**